### PR TITLE
[Keyboard] Add pluto12

### PR DESCRIPTION
**Pull request by wymcg**
## Description

Adds support for the pluto12 macropad

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/psuieee/pluto12/rules.mk [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (drashna) [inline keyboards/psuieee/pluto12/rules.mk:24]:
```suggestion
```

File: keyboards/psuieee/pluto12/rules.mk [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (drashna) [inline keyboards/psuieee/pluto12/rules.mk:11]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/psuieee/pluto12/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# pluto12
+
+![pluto12](https://i.imgur.com/XYtwexo.jpg)
```
Comment (fauxpark) [inline keyboards/psuieee/pluto12/readme.md:3]:
```suggestion
![pluto12](https://i.imgur.com/XYtwexoh.jpg)
```

File: keyboards/psuieee/pluto12/rules.mk [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/psuieee/pluto12/rules.mk:15]:
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by wymcg** [comment]
@drashna @fauxpark is there anything else you all need from me? Thanks!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**